### PR TITLE
Milestone 8 of my project

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
**In this project;**

- I created another js file 
- Created my projects dynamically in js file
- Used for loops to loop through the objects and arrays.

You can leave a comment 
Thanks

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -672,7 +736,43 @@ input.form-control::placeholder {
...
+
+  .button {
+    display: flex;
+    gap: 30px;
+    width: 300px;
+  }
+
+  .dummy-text {
+    /* width: 700px; */
+    font-weight: 400;
```
Comment (mahbubul14) [inline style.css]: - [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Please remove the inline comments from line 768.

File: modal.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    title: 'Tonic',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+    imageLinks: './img/snapshoot-portfolio.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back End Dev',
+    year: '2022',
+    company: 'Canopy',
+    live: 'https://dinma-faith.github.io/Dinma-Portfolio.github.io/',
+    source: 'https://github.com/Dinma-Faith/Dinma-Portfolio',
+  },
```
Comment (mahbubul14) [inline modal.js:14]: - [ ] Currently, your design is not matched with the Figma design (Desktop version). See the below screenshot about the space on the left, right, and bottom also. Please fix that.

- [ ] When it popup, the background should be blurred. Please work on that for both mobile and desktop versions. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
d2 | d1

- [ ] When clicking on the see project button,  the navbar is still there. Check the image below and fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
m1 | m2

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Descriptive PR title & description.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by aimemalaika — APPROVED** [review]
Hi @Dinma-Faith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**